bin/export_atlas_ebeye_xml.pl

```diff
@@ -288,6 +288,34 @@ sub fetch_degenes_experiments_contrasts_from_solrdb {
   return $geneIDs2expAccs2contrastIDs;
 }
 
+## fetch json formatted result for experiments and its titles from WebAPI.
+sub fetch_experiment_title_from_webapi {
+
+    my ( $expAcc, $logger ) = @_;
+
+    my $url = "https://wwwdev.ebi.ac.uk/gxa/json/experiments";
```

Comment:
Do we really hard-code URIs like this rather than storing in environment variables?